.github/workflows/ubuntu.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.13', '3.12', '3.11', '3.8', '3.9', '3.8']
+        python-version: ['3.13', '3.12', '3.10', '3.8']
```

Comment:
Dod you mean to change 38 to 3.9?

Reply:
I was saying to keep the even versions, plus the last one (if odd).

Reply:
3.8 is the version supported by the last version of Pyston.

Reply:
I think we should drop Pyston then in its standalone form. There is still a pyston-lite plugin. 

I don't see a reason to drop version  that end in an odd number. Python does not say that minor numbers that end in odd numbers have any less status than those with even numbers. Dropping 3.8 makes sense because the Python language drifts and it is extra effort to support older Pythons.

Reply:
It is a pitty because Pyston is much more faster than CPython and Pyston-lite, but OK. In any case, we still are compatible with 3.8